docs/gatsby-config.js

```diff
@@ -25,8 +25,10 @@ module.exports = {
             'api-reference'
           ],
           Tutorial: [
-            'tutorial/tutorial_0',
-            'tutorial/tutorial_1'
+            'tutorial/tutorial-introduction',
+            'tutorial/tutorial-create-project',
+            'tutorial/tutorial-obtain-schema',
+            'tutorial/tutorial-execute-query'
```

Comment:
i assume these are semantic instead of numbered for SEO reasons?

Reply:
Yep, SEO plus more reader-friendly URLs